opacus/privacy_engine.py

```diff
@@ -57,7 +52,7 @@ def __init__(self, accountant: str = "rdp", secure_mode=False):
             ...         optimizer.virtual_step()  # this will call privacy engine's virtual_step()
 
         """
-        self.accountant = self._prepare_accountant(mechanism=accountant)
+        self.accountant = get_accountant(mechanism=accountant)
```

Comment:
I thought the plan was to move this to `make_private` similar to optimizer?

Reply:
I would disagree - keeping accountant across multiple calls to `make_optimizer` is valuable and allows you to calculate overall epsilon over multi-stage training (e.g. if you want to change batch_size in the middle of your training)

Reply:
Good point!